File: src/main/java/com/alex/Hello.java
package com.alex;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("hello world..");



//        new Person().hello();

        Person p = new Person();
        p.hello();

        System.out.println(p.bmi());


        p.height = 1.7f;
        p.weight = 66.5f;
        p.bmi();


    }
}

File: src/main/kotlin/com/alex/Hello.kt
package com.alex

fun main() {
    println("Hello kotlin..")
    Human().hello()

    val h = Human()
    h.hello()

//    變數
/*
    val 不可變的
    var 可變的
 */

    var age = 19;
//  自動推斷
    var age2: Int = 20;
    var weight : Float = 66.5f;
    var name : String;
    name = "Alex"
    println(name)





}

class Human(){
    fun hello() {
        println("Hello human")
    }
}